Example/Example Swift/ComponentAPIDocumentsService.swift

```diff
@@ -57,36 +55,24 @@ final class ComponentAPIDocumentsService: ComponentAPIDocumentServiceProtocol {
     func resetToInitialState() {
         partialDocuments.removeAll()
         analysisCancellationToken = nil
-        compositeDocument = nil
+        document = nil
     }
     
     func update(imageDocument: GiniImageDocument) {
-        partialDocuments[imageDocument.id]?.info.rotationDelta = Int32(imageDocument.rotationDelta)
+        partialDocuments[imageDocument.id]?.info.rotationDelta = imageDocument.rotationDelta
     }
     
-    func sendFeedback(with updatedExtractions: [String: Extraction]) {
-        giniSDK.sessionManager
-            .getSession()
-            .continueWith(block: sessionBlock())
-            .continueOnSuccessWith(block: { _ in
-                return self.giniSDK
-                    .documentTaskManager?
-                    .update(self.compositeDocument,
-                            updatedExtractions: updatedExtractions,
-                            cancellationToken: nil)
-            })
-            .continueWith(block: { (task: BFTask?) in
-                if let error = task?.error {
-                    let id = self.compositeDocument?.documentId ?? ""
-                    let message = "❌ Error sending feedback for document with id: \(id) error: \(error)"
-                    print(message)
-                    
-                    return nil
-                }
-                
+    func sendFeedback(with updatedExtractions: [Extraction]) {
+        guard let document = document else { return }
+        documentService.submitFeedback(for: document, with: updatedExtractions) { result in
+            switch result {
+            case .success:
                 print("🚀 Feedback sent with \(updatedExtractions.count) extractions")
-                return nil
-            })
+            case .failure(let error):
+                let message = "❌ Error sending feedback for document with id: \(document.id) error: \(error)"
```

Comment:
I don't see a benefit in having a local var for that.

Reply:
It was like that because it was a longer line before. Now it makes sense to print directly 👍